src/composer/scenes/Composer/Composer.tsx

```diff
@@ -598,6 +607,39 @@ class Composer extends React.Component<IProperties, IState> {
     });
   }
 
+  /**
+   * Returns component by id
+   * @param id
+   * @return {}
+   */
+  public getComponentById(id: number, cont?: IContent) {
```

Comment:
```suggestion
  public getComponentById(id: number, content?: IContent) {
```

It's better to use a whole name of a variable in the function definition and then in function body use shortcut like `cont` if you need it ;)